variables.tf

```diff
@@ -129,6 +129,24 @@ variable "remote_metastores" {
   default     = []
 }
 
+#list of maps, example: [ {bastion-host="10.x.x.x", metastore-host="10.x.x.x", port="9083", prefix="pre1", user="my-unix-user", mapped-databases="test1,test2"}, {bastion-host="10.x.x.x", metastore-host="10.x.x.x", port="9083", prefix="pre1", user="my-unix-user", writable-whitelist="db1,test", mapped-databases="test1,test2"} ]
+variable "ssh_metastores" {
+  description = "List of federated Metastores to connect over SSH via Bastion."
```

Comment:
The comments I made above for the README apply to this file too (I guess this is where it's actually coming from?)